fe/lib/components/Person/SpecifiedPersonForm.tsx

```diff
@@ -0,0 +1,145 @@
+import { Button, Dropdown, Stack, TextField } from 'braid-design-system';
+import React from 'react';
+import { Controller, useForm } from 'react-hook-form';
+
+import { SpecifiedPersonInput } from '../../types/seek.graphql';
+
+type RoleCode = 'HiringManager' | 'Recruiter';
+
+interface Props {
+  onCreate: (person: SpecifiedPersonInput) => void;
+}
+
+interface SubmitData {
+  roleCode: RoleCode;
+  givenName: string;
+  familyName: string;
+  email?: string;
+  phone?: string;
+}
+
+const mapFormDataToMutationInput = (
+  formData: SubmitData,
+): SpecifiedPersonInput => ({
+  roleCode: formData.roleCode,
+  communication: {
+    email: formData.email
+      ? [
+          {
+            address: formData.email,
+          },
+        ]
+      : [],
+    phone: formData.phone
+      ? [
+          {
+            formattedNumber: formData.phone,
+          },
+        ]
+      : [],
+  },
+  name: {
+    given: formData.givenName,
+    family: formData.familyName,
+    formattedName: `${formData.givenName} ${formData.familyName}`,
+  },
+});
+
+export const SpecifiedPersonForm = ({ onCreate }: Props) => {
+  const { handleSubmit, control, errors } = useForm<SubmitData>();
+
+  return (
+    <form
+      onSubmit={handleSubmit((formData) =>
+        onCreate(mapFormDataToMutationInput(formData)),
+      )}
+    >
+      <Stack space="small">
+        <Controller
+          render={(formProps) => (
+            <Dropdown
+              id="roleCode"
```

Comment:
These get used as unique identifiers in the document so we may want to namespace them.